### PR TITLE
Add a part about solutions

### LINKED ISSUE #1683: Add a part about solutions
I think we need an additional menu item called solutions(EN) / Lösungen(DE) / Oplossingen(NL)

For the German website I would want the following pages under that menu:
- Gesundheitswesen
- Verkehr und Logistik
- Performance Support
- E-Learning

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Close #1683

- [x] Add a solutions page
- [x] Add to navigations
- [x] Add a health care page
- [x] Fix metadata
- [x] Add content to health care page
- [x] Add content to index page 
- [x] Fix screenshots
- [x] Fix meta-description for solutions page

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
> Add form partial with local variables

Start of new approach of forms. There could be just one partial with local variables for variants.

**Comment by RonaldDijkstra** [comment]
@sn3p this is ready. I know it's kinda big so I want to kickstart the review, so: 

Deploy preview: 

**Index:** 
https://staging.defacto.nl/loesungen/ 

**Detail:** 
https://staging.defacto.nl/gesundheitswesen 

2 things that are in this PR but just a little bit in scope:

1. Implement partial/form with local variables to populate the form. This is a first, but is meant for all other forms too
2. Globalized the section "on-page-navigation", it's on multiple pages so it's a component from now on

**Comment by RonaldDijkstra** [comment]
CircleCI build fails because of html-proofer not finding the pages (it checks the open graph url, which are the full_url/page_path) 

```bash
- ./build/gesundheitswesen/index.html
  *  External link https://www.defactolearning.de/gesundheitswesen/ failed: 404 No error
- ./build/loesungen/index.html
  *  External link https://www.defactolearning.de/loesungen/ failed: 404 No error
```

so nothing wrong here.

**Comment by RonaldDijkstra** [comment]
@EasterPeanut do you maybe have some time to review this beast?

File: source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown [hunk 1]
```diff
@@ -23,6 +23,7 @@ facts:
   url: http://www.evangelischeskrankenhaus.de
   since: 2015
   employees: 1200
+tags: "healthcare"
 ---
 
 __Das grenzüberschreitende Gemeinschaftsprojekt im Rahmen der Medizinerausbildung der European Medical School ist ein gutes Beispiel für eine gelungene deutsch-niederländische Kooperation des Evangelischen Krankenhauses Oldenburg (EV). Eine weitere gute Zusammenarbeit bietet sich zwischen dem EV und Defacto an: Die Lösungen “CAPP Talent Management”, “CAPP Talent Management” und der “CAPP Qualitätspass” können vor dem Hintergrund der Vision des lebenslangen Lernens den dortigen Kulturwandel gemeinsam gestalten. Das Krankenhaus verwendet bereits seit 2015 ein modernes Talentmanagement System und entwickelt eigene E-Learning Kurse.__
```
Comment (EasterPeanut) [inline source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown]: CAPP Learning Management I think? 

“CAPP Talent Management”, “CAPP Talent Management” und der “CAPP Qualitätspass”

**Comment by EasterPeanut** [comment]
Came across some styling issues while going through staging:

Gesundheidswesen on smaller views:

Screenshot 2019-03-11 at 20 10 34
Screenshot 2019-03-11 at 20 18 06

No spacing between "kunden" for smaller views:

Screenshot 2019-03-11 at 20 12 33

Title left outline on product pages on smaller views:
Screenshot 2019-03-11 at 20 17 03

**Comment by EasterPeanut** [comment]
How can I get to visit the 'gezondheidswezen' page locally? Wanted to try something

**Comment by RonaldDijkstra** [comment]
git fetch and then checkout?

**Comment by RonaldDijkstra** [comment]
https://public.3.basecamp.com/p/rJEU82ULTV7BXpL43jGBVRe7 

rake serve:de 
or
LOCALE=de bundle exec middleman

**Comment by RonaldDijkstra** [comment]
Great review

**Comment by RonaldDijkstra** [comment]
> No spacing between "kunden" for smaller views:

#1729

File: source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [reply source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown]: Sharp as a razor

Comment (RonaldDijkstra) [reply source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown]: Wow this is in the user case

File: source/localizable/partials/_featured-testimonial.erb [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <section class="testimonials featured-testimonials">
-  <div class="headline">
-    <h2><%= t("testimonials.title") %></h2>
-  </div>
+  <% unless current_page.locale_root_path =~ /gezondheidswezen.html/ %>
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/localizable/partials/_featured-testimonial.erb]: Actually unneeded. Just import the "testimonials" partial instead.

**Comment by RonaldDijkstra** [comment]
Title left outline on product pages on smaller views: separate issue #1730

**Comment by RonaldDijkstra** [comment]
@EasterPeanut fixed all things or made issues for them

**Comment by RonaldDijkstra** [comment]
https://staging.defacto.nl/loesungen
https://staging.defacto.nl/gesundheitswesen

**Comment by isabelamy** [comment]
Some content changes:

1. Testimonial of Anne Monier has to be replaced with Alke Stade
2. E-Learning link moet naar https://www.defactolearning.de/e-learning-im-gesundheitswesen/
3. CAPP Learning & Talent Management link moet naar https://www.defactolearning.de/capp-learning-management/
4. Er is nog een spacing issue --> ...Organisation des Kurssystems und **der Seminarverwaltung**.
5. Er staat twee keer een stuk over 'Seminarverwaltung und Kursverwaltung'. De tweede keer moet het een andere tekst zijn zie https://defactosoftwarebv.sharepoint.com/:w:/r/_layouts/15/Doc.aspx?sourcedoc=%7B372BB058-8D17-4789-A215-C71258F44E0F%7D&file=gesundheitswesen%20seite.docx&action=default&mobileredirect=true
6. bij de logos missen nog Medius en MHH
7. ik zou bij het deel 'Mehr zum Gesundheitswesen' de links groter willen hebben of ook tegels, maar dat kan op een ander moment.

**Comment by EasterPeanut** [comment]
Evangelisches Krankenhaus Oldenburg Case has a duplicate in Produkte & Dienste:
Screenshot 2019-03-14 at 14 59 27

**Comment by RonaldDijkstra** [comment]
#1744

**Pull request merged**